modules/apps/frontend-theme/kyoung-liferay-theme/src/templates/navigation.ftl

```diff
@@ -0,0 +1,74 @@
+<nav class="${nav_css_class}" id="navigation" role="navigation">
+	<h1 class="hide-accessible"><@liferay.language key="navigation" /></h1>
+
+	<ul aria-label="<@liferay.language key="site-pages" />" role="menubar">
+		<#list nav_items as nav_item>
+			<#assign
+				nav_item_attr_has_popup = ""
+				nav_item_css_class = ""
+				nav_item_layout = nav_item.getLayout()
+			/>
+
+			<#if nav_item.isSelected()>
+				<#assign
+					nav_item_attr_has_popup = "aria-haspopup='true'"
+					nav_item_css_class = "selected"
+				/>
+			</#if>
+
+			<li class="${nav_item_css_class}" id="layout_${nav_item.getLayoutId()}" role="presentation">
+				<#if nav_item.hasChildren()>
+					<a aria-labelledby="layout_${nav_item.getLayoutId()}" ${nav_item_attr_has_popup} href="${nav_item.getURL()}" ${nav_item.getTarget()} role="menuitem"><span><@liferay_theme["layout-icon"] layout=nav_item_layout /> ${nav_item.getName()}<i class="fa fa-angle-down"></i></span></a>
+                    <#else>
+					<a aria-labelledby="layout_${nav_item.getLayoutId()}" ${nav_item_attr_has_popup} href="${nav_item.getURL()}" ${nav_item.getTarget()} role="menuitem"><span><@liferay_theme["layout-icon"] layout=nav_item_layout /> ${nav_item.getName()}</span></a>
+                </#if>
+				<#if nav_item.hasChildren()>

```

Comment:
There should be a line break between Freemarker statements for better readability and clarity.